CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+
+# 5.13.0 / 4.13.0
```

Comment:
Move this under `5.13.1 / 4.13.1` as that version is not published yet.

Reply:
done.

Reply:
Um, I meant to merge it with the previous section, you have two with same versions there now :)

Reply:
thanks for your help, does this work now? @FredyC

Reply:
Ughhh ... I said to move this under 5.13.1 / 4.13.1, not to make another version :)

Reply:
I think I got it this time :P @FredyC